packages/alchemy/src/gas-manager.ts

```diff
@@ -4,18 +4,15 @@ import {
   arbitrumSepolia,
   base,
   baseSepolia,
-  fraxtal,
-  fraxtalTestnet,
   mainnet,
+  sepolia,
   optimism,
   optimismSepolia,
   polygon,
   polygonAmoy,
   polygonMumbai,
-  sepolia,
-  zora,
-  zoraSepolia,
-} from "viem/chains";
+} from "@alchemy/aa-core";
+import { fraxtal, fraxtalTestnet, zora, zoraSepolia } from "viem/chains";
```

Comment:
+1 to dennis's comment about consolidating these into aa-core

Reply:
Added to aa-core without alchemy url